content/implementation-path.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Implementation Path"
+---
+
+The work to be done in support of naming changes is generally low intensity, but can have consequences. We suggest the following:
+
+1. Divide changes into internal changes, customer-facing changes, and build/tooling changes
+2. Assess the scope of changes to make in each of these categories using simple tools like `grep` searches.
+3. Make changes incrementally, providing concurrent support
+4. Deprecate old terminology
+
+We also suggest doing these changes in order:
+
+1. Internal changes 
+2. Customer-facing changes
+3. Build/tooling changes
+
+## Internal changes
+
+### Assess the project's source code
+
+1. Do a recursive search of the project source code and documentation (e.g., `grep -Ri XYZ .`), where `XYZ` is the old term. 
+2. Review the results to identify changes, and do them incrementally. Do not edit generated files. Instead, fix the source files and regenerate the files. 
+
+### Check the results 
+
+There are many ways to do a search and replace of files once you have determined that the changes are correct. A general command line command on POSIX systems to recursively rename `foo` into `bar` (when GNU sed is used) for filenames ending in `.txt` is:
+
+```bash
+find . -type f -name '*.txt' -exec sed -i'' -e 's/foo/bar/g' {} +
+```
+On MacOS and other BSD-based systems, put a space after `-i` (GNU sed requires no space while BSD sed requires a space).
+
+Change `find .` to `find NAME` to start at directory `NAME` instead of the current directory. This is a case-sensitive match that replaces the original term even within other terms/phrases.
+
+### Add concurrent support 
+
+Where practical add support for new names while supporting old names. This allows projects to change incrementally instead of requiring changes to occur simultaneously across all related projects and organizations. It also reduces the risk of system failure.
+
+For projects that have automated test suites, add tests to ensure that either both old and new names work, or that at a minimum new names work. This allows end users to incrementally transition with confidence.
+
+In some projects this work will take significant time, be gracious to those doing the work, and be understanding that it will take time to do all this coordination.
+ 
+## Customer-facing changes
+
+Changes to customer facing terminology like URLs are harder to deal with. Run through the steps for internal changes in addition to the following two points.
+
+### URLs, URL fragments, command and API names
```

Comment:
```suggestion
### URLs, URL fragments, command, and API names
```

Reply:
FYI: i'm taking this (and other `suggest` suggestions) but I'm going to add them as a commit via my local and dismiss these as I go :)